.github/workflows/release.yaml

```diff
@@ -1,88 +1,34 @@
-        description: WASM Shim version
-        default: 0.0.0
-        type: string
-      consolePluginImageURL:
-        description: ConsolePlugin image URL
-        default: "quay.io/kuadrant/console-plugin:latest"
-        type: string
-      prerelease:
-        description: Is the release a pre-release?
-        required: false
-        type: boolean
+  pull_request:
+    types:
+    - closed
+    branches:
+      - 'release-v[0-9]+.[0-9]+'
 
 jobs:
-  build:
+  if_merged:
+    if: github.event.pull_request.merged == true
     name: Release operator
     runs-on: ubuntu-latest
     steps:
+      # TODO: not sure if the `Install gettext-base`is still required
       - name: Install gettext-base
         run: |
           sudo apt-get update
           sudo apt-get install -y gettext-base
-      - name: Set up Go 1.21.x
-        uses: actions/setup-go@v4
-        with:
-          go-version: 1.21.x
-        id: go
       - name: Checkout code at git ref
         uses: actions/checkout@v4
-        with:
-          ref: ${{ inputs.gitRef }}
-          token: ${{ secrets.KUADRANT_DEV_PAT }}
-      - name: Create release branch
-        if: ${{ !startsWith(inputs.gitRef, 'release-v') }}
+      - name: Set envoironment variables
         run: |
-          git checkout -b release-v${{ inputs.kuadrantOperatorVersion }}
-      - name: Prepare release
-        run: |
-          make prepare-release \
-          VERSION=${{ inputs.kuadrantOperatorVersion }} \
-          AUTHORINO_OPERATOR_VERSION=${{ inputs.authorinoOperatorVersion }} \
-          LIMITADOR_OPERATOR_VERSION=${{ inputs.limitadorOperatorVersion }} \
-          DNS_OPERATOR_VERSION=${{ inputs.dnsOperatorVersion }} \
-          WASM_SHIM_VERSION=${{ inputs.wasmShimVersion }} \
-          RELATED_IMAGE_CONSOLEPLUGIN=${{ inputs.consolePluginImageURL }}
-      - name: Commit and push
-        uses: stefanzweifel/git-auto-commit-action@v5
-        with:
-          commit_message: "Prepared release v${{ inputs.kuadrantOperatorVersion }}"
-          commit_user_name: "github-actions[bot]"
-          commit_user_email: "github-actions[bot]@users.noreply.github.com"
-          branch: release-v${{ inputs.kuadrantOperatorVersion }}
-          create_branch: true
-          tagging_message: v${{ inputs.kuadrantOperatorVersion }}
-          commit_options: '--signoff'
+          bash ./hack/release/load_envvar.sh
       - name: Create release
         uses: softprops/action-gh-release@v1
         with:
-          name: v${{ inputs.kuadrantOperatorVersion }}
-          tag_name: v${{ inputs.kuadrantOperatorVersion }}
-          body: "**This release enables installations of Authorino Operator v${{ inputs.authorinoOperatorVersion }}, Limitador Operator v${{ inputs.limitadorOperatorVersion }}, DNS Operator v${{ inputs.dnsOperatorVersion }}, WASM Shim v${{ inputs.wasmShimVersion }} and ConsolePlugin ${{ inputs.consolePluginImageURL }}**"
+          name: v${{ kuadrantOperatorVersion }}
```

Comment:
where is `kuadrantOperatorVersion` being set? if it is an env var, it should be `env.kuadrantOperatorVersion`

Reply:
I will look at this.

Reply:
Yes, they should be prefixed with `env.`